Gemfile.lock

```diff
@@ -81,4 +84,4 @@ DEPENDENCIES
   rubocop-rspec
 
 BUNDLED WITH
-   1.17.3
+   2.5.17
```

Comment:
this has been updated to use a modern bundler, and be compatible with ruby 3.1+. many of the transitive dependences have changed simply after running `bundle install`, but they won't affect financeit's gemfile, other than that we eventually hope to see `rest-client` disappear.